src/lib/api/services/upload.service.ts

```diff
@@ -8,25 +8,8 @@ import mime from 'mime';
 import Spinner from '../../../helper/spinner';
 
 export default {
-    startUpload: async (data, namespace) => {
-        try {
-            const axiosOption = Object.assign(
-                {},
-                {
-                    data: data,
-                },
-                getCommonHeaderOptions(),
-            );
-            const res = await ApiClient.post(
-                URLS.START_UPLOAD_FILE(namespace),
-                axiosOption,
-            );
-            return res;
-        } catch (error) {
-            throw error;
-        }
-    },
-    uploadFile: async (filepath, namespace, file_name = null) => {
+
+    uploadFile: async (filepath, namespace, file_name = null, mimeType = null) => {
```

Comment:
Keeping default values here suggests they are acceptable, can we remove them ?

Reply:
This function was designed this way. These are just preferential values. So if I pass `file_name` - that is used, otherwise we extract the file name from the actual file that is getting uploaded. Same for mimeType. Basically, If i dont pass file_name and mimeType, the function takes them from the actual file